libs/styles/src/lib/scss/components/_steps.scss

```diff
@@ -347,5 +356,52 @@
         @include cvi-screenreader-text;
       }
     }
+
+    #{$base}.is-scrollable {
+      & #{$base}__scroll-container {
+        @include cvi-breakpoint-down(sm) {
+          display: grid;
+        }
+
+        display: flex;
+        width: 100%;
+        overflow-x: auto;
+        scrollbar-width: none;
+        scroll-behavior: smooth;
+      }
+    }
+
+      & #{$base}__scroll-button {
+        @include cvi-breakpoint-down(sm) {
+          display: none;
+        }
+        @include cvi-breakpoint-up(sm) {
+        position: absolute;
+        height: 72px;
```

Comment:
With more than 3 lines of text in any item, the display gets broken: 
![image](https://github.com/e-gov/cvi/assets/1833029/da3f65c4-5959-4be3-b18e-639bbce5db3e)

Maybe this could be fixed by introducing an extra div wrapping the `cvi-steps__scroll-container`. It would have `position:relative` on it. The scroll buttons could be moved to that container, alongside the  `cvi-steps__scroll-container`. Then they could have `height: 100%` instead of hardcoded height, and would get taller according to the size of the steps titles.

Reply:
Used your solution, but needed to calculate the height. Navigation bar has bottom padding 24px. Tried to add padding, but with relative position, it don't work. So best option what I found.